### DISCUSSION TITLE
Filter in search by supported languages

### DISCUSSION THREAD

**Discussion by Xiaohuang257** [modrinth/.github#20] [Feature requests]
### Is your feature request related to a problem? Please describe.

Add a mod language choose.

### Additional context

![image](https://user-images.githubusercontent.com/81309333/176250273-2cf9a91d-1ccc-4fc6-a57a-05778078f661.png)

**Comment by Xiaohuang257** [comment]
I'm sorry I don't speak English
[DeepL]

**Comment by triphora** [comment]
你好!
谢谢你的建议。这是我们正在计划做的事情。在 modrinth/labrinth#359 已经有一个问题。前台本身也将很快有一个翻译，这将在 modrinth/knossos#345 被合并后完成。你可以在 https://rewrite.modrinth.com 上看到一个预览。

Duplicate of modrinth/labrinth#359

**Comment by Xiaohuang257** [comment]
> 你好! 谢谢你的建议。这是我们正在计划做的事情。在 [modrinth/labrinth#359](https://github.com/modrinth/labrinth/issues/359) 已经有一个问题。前台本身也将很快有一个翻译，这将在 #345 被合并后完成。你可以在 https://rewrite.modrinth.com 上看到一个预览。
> 
> [modrinth/labrinth#359](https://github.com/modrinth/labrinth/issues/359)的副本

啊，有国人的话就好说了，其实我想说的并不是页面的语言，而是模组或整合包的。
你可以查看模组或整合包支持哪些语言，并且可以通过分类更快的找到包含该语言的模组或整合包

**Comment by triphora** [comment]
啊，这确实有道理。我将相应地修改标题并重新开放。

P.S. 我实际上是在用DeepL 来翻译这个 :D

**Comment by Xiaohuang257** [comment]
> 啊，这确实有道理。我将相应地修改标题并重新开放。
> 
> P.S. 我实际上是在用DeepL 来翻译这个 :D

XP
